udata_hydra/cli.py

```diff
@@ -252,19 +252,13 @@ async def migrate(skip_errors: bool = False, dbs: list[str] = ["main", "csv"]):
 
 
 @cli
-async def purge_checks(limit: int = 2):
-    """Delete past checks for each resource_id, keeping only `limit` number of checks"""
-    q = "SELECT resource_id FROM checks GROUP BY resource_id HAVING count(id) > $1"
+async def purge_checks(days_until: int = 60):
```

Comment:
I'm not sure about `days_until` wording. How about `retention_days`? (It's the wording we're using for [harvest jobs retention policy](https://github.com/opendatateam/udata/blob/c7bd1c28c399552164ccd5d448805c76e8bfeaa7/udata/harvest/actions.py#L168))

Reply:
Yep, that sounds more clear. Let me update that

Reply:
Done